src/modules/api/pipeline-task/pipelines/db_op.py

```diff
@@ -1,30 +1,27 @@
-import os
-import logging
-
 from caendr.utils import monitor
 from caendr.models.task import DatabaseOperationTask
 from caendr.models.datastore import DatabaseOperation
 from caendr.services.cloud.lifesciences import start_pipeline
 from caendr.models.lifesciences import ServiceAccount, VirtualMachine, Resources, Action, Pipeline, Request
-from caendr.utils.json import get_json_from_class
```

Comment:
this is being removed, but I'll add this note here anyway for future reference -- 
API semantic note -- `get_json_from_class` is usually implemented as a method `#to_json()` on the class that it applies to.